Assets/HoloToolkit/Input/Scripts/Focus/FocusManager.cs

```diff
@@ -477,16 +528,20 @@ private void RaycastPhysics(PointerData pointer, LayerMask[] prioritizedLayerMas
 
         private void RaycastUnityUI(PointerData pointer, LayerMask[] prioritizedLayerMasks)
         {
-            GetPointerEventData();
+            pointer.UnityUIPointerData.Clear();
```

Comment:
Now that we're handling our ui pointer data for each pointer separately we should remove this and clear the data after input up/clicked.